src/CommitmentsV2/SFA.DAS.CommitmentsV2/Data/AccountConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using SFA.DAS.CommitmentsV2.Models;
+
+namespace SFA.DAS.CommitmentsV2.Data
+{
+    public class AccountConfiguration : IEntityTypeConfiguration<Account>
+    {
+        public void Configure(EntityTypeBuilder<Account> builder)
+        {
+            builder.Property(a => a.Id).ValueGeneratedNever();
+            builder.Property(a => a.HashedId).IsRequired().HasColumnType("char(6)");
+            builder.Property(a => a.PublicHashedId).IsRequired().HasColumnType("char(6)");
```

Comment:
Shouldn't these be nchar(6)?

Reply:
no they can only be char in the DB. The hashing values are limited to A-Z1-9

Reply:
changed